### PR TITLE
add time granularity to redshift

### PR DESCRIPTION
**Pull request by CyrilLeMat**

### CODE REVIEW

**Review by xrmx — COMMENTED** [review]
These looks exactly like the inherited postgresql one. I'm not sure the patch is needed

**Comment by CyrilLeMat** [comment]
yes it's exactly like the postgresql time grains. I added it because on the 0.20, I'm not able anymore to use time grains on redshift. If it's supposed to inherit from postgresl, then it's a bug. is it supposed to do so?

File: superset/db_engine_specs.py [hunk 1]
```diff
@@ -866,6 +866,18 @@ def convert_dttm(cls, target_type, dttm):
 
 class RedshiftEngineSpec(PostgresEngineSpec):
     engine = 'redshift'
+    
+    time_grains = (
```
Comment (mistercrunch) [inline superset/db_engine_specs.py]: isn't this inherited from `PostgresEngineSpec`? Looks like a copy-paste of exactly what is inherited...

**Comment by mistercrunch** [comment]
Did your patch here fix the "bug"? What are the symptoms, is the dropdown empty?

**Pull request closed without merging**